chia/wallet/wallet_rpc_api.py

```diff
@@ -1297,22 +1302,21 @@ async def _get_wallet_balance(self, wallet_id: uint32) -> dict[str, Any]:
                 assert isinstance(wallet, CRCATWallet)
                 wallet_balance["pending_approval_balance"] = await wallet.get_pending_approval_balance()
 
-        return wallet_balance
+        return BalanceResponse.from_json_dict(wallet_balance)
```

Comment:
feels like we shouldn't need to convert to json, modify, then back?

Reply:
we'll follow up together maybe with https://github.com/Chia-Network/chia-blockchain/pull/19861/files